packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -4,26 +4,24 @@
 import { AccessToken, TokenCredential, GetTokenOptions } from "@azure/identity";
```

Comment:
Also update the interface in teamsUserCredential.ts file

Reply:
Fixed. Thx!